modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -688,8 +688,8 @@
     <string name="profile_cancel_confirm_item_folders">Your folders will be removed and their contents will move back to the Podcasts screen.</string>
     <string name="profile_cancel_confirm_item_uploads">All files uploaded to your Pocket&#160;Casts account will be deleted (but downloaded files on your mobile devices will remain).</string>
     <string name="profile_cancel_confirm_item_web_player">You will no longer be able to access Pocket&#160;Casts using your web browser, or desktop computer.</string>
-    <string name="profile_cancel_confirm_stay_button_title">Actually, I want to stay"</string>
-    <string name="profile_cancel_confirm_cancel_button_title">Yes, cancel my subscription"</string>
```

Comment:
This seems to be confusing the translations, the quote is safe to be removed. 

![Screenshot_20230731_230705](https://github.com/Automattic/pocket-casts-android/assets/308331/2eb3c714-d47f-4273-aacb-5cbeacb29022)